src/aics-image-viewer/components/GlobalVolumeControls.tsx

```diff
@@ -99,6 +122,7 @@ export default class GlobalVolumeControls extends React.Component {
     return this.createSliderRow(config);
   }
 
+  // TODO: `config` object has no type annotation; this component deserves some DRY edits that may reduce it out
```

Comment:
once we are fully "typescriptified" we need to have this be a `lint` or `typeCheck` violation.

Reply:
Agreed. The intent of this comment (which I did a poor job expressing in few words) is "I would have written a type for `config`, but I don't think it has a good reason to exist in the first place, but if it's going to be removed it'll be as part of a larger refactor of this component that's beyond the scope of my current task."